src/daemon/org.projectatomic.rpmostree1.policy

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE policyconfig PUBLIC
+ "-//freedesktop//DTD PolicyKit Policy Configuration 1.0//EN"
+ "http://www.freedesktop.org/standards/PolicyKit/1.0/policyconfig.dtd">
+<policyconfig>
+
+  <vendor>Project Atomic</vendor>
+  <vendor_url>https://www.projectatomic.io/</vendor_url>
+  <icon_name>package-x-generic</icon_name>
+
+  <action id="org.projectatomic.rpmostree1.install-uninstall-packages">
+    <description>Install and remove packages</description>
+    <message>Authentication is required to install and remove software</message>
+    <icon_name>package-x-generic</icon_name>
+    <defaults>
+      <allow_any>auth_admin</allow_any>
+      <allow_inactive>auth_admin</allow_inactive>
+      <allow_active>auth_admin_keep</allow_active>
+    </defaults>
+  </action>
+
+  <action id="org.projectatomic.rpmostree1.install-local-packages">
```

Comment:
The reason why I split local package installing out was that I think it's inherently a more unsafe operation than installing from a repo. For instance, I think it might make sense to make installing from a repo work without password prompt in Fedora Workstation, but when someone wants to install a random package downloaded from the internet (and not signed with our gpg keys etc), we should ask for confirmation.

I think it's fine that there's only one uninstall action, I don't think there's a need for separate uninstall-local-packages